makefile.old

```diff
@@ -0,0 +1,120 @@
+#!make
```

Comment:
This is the existing make file, moved to makefile.old for backwards compatability until migration is fully complete. 

The github actions are updated to use this file. For local, use this file.